gpflow/multioutput/features.py

```diff
@@ -167,7 +178,7 @@ def Kuu(feat, kern, *, jitter):
     return Kmm + jittermat
 
 
-@dispatch(SeparateIndependentMof, (SeparateIndependentMok, SeparateMixedMok))
+@dispatch((SeparateIndependentMof,MixedKernelSeparateMof), (SeparateIndependentMok, SeparateMixedMok))
```

Comment:
Is this supposed to be an implementation for the outer product of (SeparateIndependentMof,MixedKernelSeparateMof) and (SeparateIndependentMok, SeparateMixedMok)? I.e. is this correct for Kuu(SeparateIndependentMof, SeparateMixedMok) and Kuu(MixedKernelSeparateMof, SeparateIndependentMok)?

Reply:
Yes. Prior to my addition it was already `SeparateIndependentMof, (SeparateIndependentMok, SeparateMixedMok)` which simply zips along both the feature list and kernel list and stack into shapes `LxMxN` for `Kuf` and `LxMxM` for `Kuu`. The same behaviour is required for any scenario where there are multiple features and multiple kernels, which is true of any combination of  `((SeparateIndependentMof,MixedKernelSeparateMof), (SeparateIndependentMok, SeparateMixedMok))`.